environments/kolla/files/overlays/horizon/custom_local_settings

```diff
@@ -1,4 +1,11 @@
-# customer specific configuration
+WEBSSO_ENABLED = True
 
-# SESSION_TIMEOUT = 7200
-# OPENSTACK_ENABLE_PASSWORD_RETRIEVE = True
+WEBSSO_KEYSTONE_URL = "http://192.168.16.10:5000/v3"
+WEBSSO_CHOICES = (
+    ("credentials", "Keystone Credentials"),
+    ("gx-scs_openid", "Authenticate via gx-scs"),
```

Comment:
`keycloak` and `Authenticate via Keycloak`

Reply:
Done